### ISSUE TITLE
Motion Notifications - Camera Not Responding

### ISSUE THREAD

**Issue by stubbz00** [dgreif/ring#605]
### **Description of Problem**

Whenever I get a ring or motion notification and click on the notification my camera will immediately show "camera not responding".  Closing out the feed and reopening a live feed the camera responds as it should.  I'm not seeing anything funny in the logs that would hint at what is going on.

### **Plugin**
            "hideDoorbellSwitch": true,
            "hideCameraMotionSensor": false,
            "hideInHomeDoorbellSwitch": true,
            "hideUnsupportedServices": true,

### **Environment**
Host: Raspberry Pi 4
Node.js Version: 14.16.1
Npm Version: 7.8.0
Devices: Doorbell Pro, Wired Spotlight Cam.

**Comment by dgreif** [comment]
Are you on HOOBS or homebridge?  I'd try rebooting your homebridge server, your phone and your Home Hub (apple tv/ipad/homepod) and see if it still happens.

**Comment by stubbz00** [comment]
I’m running Homebridge.  This has been going on for the last couple of months but is intermittent.  If I click on a old notification the live view works just fine.  If I click on a notification shortly after it came through I will usually get “camera not responding”.  However, this is not always the case.  Just a few minutes ago a motion notification came through and I immediately opened the notification and the live feed came through just fine.

Could this possibly be an issue with snapshots?

**Comment by stubbz00** [comment]
I just recently updated my npm and node.js versions about 2 days ago from relatively old versions of both.  I don't believe I restarted my Raspberry Pi between these updates, which might have been still contributing to this problem.  Since rebooting rebooting the Raspberry Pi this issue has been much less frequent.  I've haven't encounter this issue at all today.    I'm going to go ahead and close this ticket.  I apologize @dgreif .
